Dalamud/Game/Gui/ContextMenu/CharacterData.cs

```diff
@@ -0,0 +1,472 @@
+    /// They're in gpose.
+    /// </summary>
+    CameraMode = 1ul << 18,
+
+    /// <summary>
+    /// They're looking for repairs.
+    /// </summary>
+    LookingForRepairs = 1ul << 19,
+
+    /// <summary>
+    /// They're looking to repair.
+    /// </summary>
+    LookingToRepair = 1ul << 20,
+
+    /// <summary>
+    /// They're looking to meld materia.
+    /// </summary>
+    LookingToMeldMateria = 1ul << 21,
+
+    /// <summary>
+    /// They're roleplaying.
+    /// </summary>
+    RolePlaying = 1ul << 22,
+
+    /// <summary>
+    /// They're looking for a party.
+    /// </summary>
+    LookingForParty = 1ul << 23,
+
+    /// <summary>
+    /// They're a sword for hire.
+    /// </summary>
+    SwordForHire = 1ul << 24,
+
+    /// <summary>
+    /// They're waiting for duty finder.
+    /// </summary>
+    WaitingForDutyFinder = 1ul << 25,
+
+    /// <summary>
+    /// They're recruiting party members.
+    /// </summary>
+    RecruitingPartyMembers = 1ul << 26,
+
+    /// <summary>
+    /// They're a mentor.
+    /// </summary>
+    Mentor = 1ul << 27,
+
+    /// <summary>
+    /// They're a PvE mentor.
+    /// </summary>
+    PvEMentor = 1ul << 28,
+
+    /// <summary>
+    /// They're a trade mentor.
+    /// </summary>
+    TradeMentor = 1ul << 29,
+
+    /// <summary>
+    /// They're a PvP mentor.
+    /// </summary>
+    PvPMentor = 1ul << 30,
+
+    /// <summary>
+    /// They're a returner.
+    /// </summary>
+    Returner = 1ul << 31,
+
+    /// <summary>
+    /// They're a new adventurer.
+    /// </summary>
+    NewAdventurer = 1ul << 32,
+
+    /// <summary>
+    /// They're the leader of an alliance.
+    /// </summary>
+    AllianceLeader = 1ul << 33,
+
+    /// <summary>
+    /// They're the leader of an alliance party.
+    /// </summary>
+    AlliancePartyLeader = 1ul << 34,
+
+    /// <summary>
+    /// They're a member of an alliance party.
+    /// </summary>
+    AlliancePartyMember = 1ul << 35,
+
+    /// <summary>
+    /// They're the leader of a party.
+    /// </summary>
+    PartyLeader = 1ul << 36,
+
+    /// <summary>
+    /// They're a member of a party.
+    /// </summary>
+    PartyMember = 1ul << 37,
+
+    /// <summary>
+    /// They're the leader of a cross-world party.
+    /// </summary>
+    PartyLeaderCrossWorld = 1ul << 38,
+
+    /// <summary>
+    /// They're a member of a cross-world party.
+    /// </summary>
+    PartyMemberCrossWorld = 1ul << 39,
+
+    /// <summary>
+    /// They're in another world.
+    /// </summary>
+    AnotherWorld = 1ul << 40,
+
+    /// <summary>
+    /// They're in a duty with you.
+    /// </summary>
+    SharingDuty = 1ul << 41,
+
+    /// <summary>
+    /// They're in a similar duty to you.
+    /// </summary>
+    SimilarDuty = 1ul << 42,
+
+    /// <summary>
+    /// They're in duty.
+    /// </summary>
+    InDuty = 1ul << 43,
+
+    /// <summary>
+    /// They're a trail adventurer.
+    /// </summary>
+    TrailAdventurer = 1ul << 44,
```

Comment:
I fixed this typo 2 days ago in client structs. (https://github.com/aers/FFXIVClientStructs/commit/8a08fa4ce28752a2040f8bfab44d8e5055e6a1ce) 😅

```suggestion
    /// <summary>
    /// They're a trial adventurer.
    /// </summary>
    TriallAdventurer = 1ul << 44,
```

Reply:
Remove this entire enum please. It’s sourced from Lumina, and I see no reason to duplicate effort.